tsdb/engine.go

```diff
@@ -60,6 +74,21 @@ func NewEngine(path string, walPath string, options EngineOptions) (Engine, erro
 	// Only bolt-based backends are currently supported so open it and check the format.
 	var format string
 	if err := func() error {
+		// if it's a dir then it's a tsm1 engine
```

Comment:
Once we have another one we can check the files in the directory. They have a magic number at the beginning for tsm1. This works for now.